src/components/Flow.web.tsx

```
@@ -266,10 +259,28 @@ function renderModal(state: FlowState, flow: UseFlowReturn, copyCurrentMessage:
       );
 
     case 'select':
+      const visibleOptions = getVisibleSelectOptions(state);
```

Biome (2.4.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)